src/main/java/edu/iastate/ece/sd/sdmay2126/runner/selenium/SeleniumRunner.java

```diff
@@ -225,15 +225,15 @@ private void executeFBARunner(Job job) throws
         new FBASeleniumInputProgrammer(driver).programInputs(job, fbaCardScope);
 
         // Next execute the application
-        new FBASeleniumApplicationExecutor(driver).executeApplication(job, fbaCardScope);
-
-        // Lastly collect results
-        job.setOutput(new FBASeleniumOutputCollector(driver).collectOutput(job, fbaCardScope));
-
-        System.out.println("Selenium Runner: Job complete!");
-
-        JobOutputWriter jobOutputWriter = new JSONJobOutputWriter();
-        jobOutputWriter.outputToFile(job, "FBAJson");
+//        new FBASeleniumApplicationExecutor(driver).executeApplication(job, fbaCardScope);
+//
+//        // Lastly collect results
+//        job.setOutput(new FBASeleniumOutputCollector(driver).collectOutput(job, fbaCardScope));
+//
+//        System.out.println("Selenium Runner: Job complete!");
+//
+//        JobOutputWriter jobOutputWriter = new JSONJobOutputWriter();
+//        jobOutputWriter.outputToFile(job, "FBAJson");
```

Comment:
Don't commit this commented out. Remove these comments (assuming they were for debugging).

Reply:
Still need to remove these commented out lines.